11ty/definitions/antisemitism.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Antisemitism
+slug: antisemitism
+defined: true
+speech: noun
+excerpt: aggression towards, discrimination, or prejudice against Jewish people.
+reading:
+   - text: "Antisemitism Education, Training &amp; Thought Leadership, from Jews for Racial &amp; Economic Justice"
+     href: https://www.jfrej.org/assets/uploads/JFREJ-Understanding-Antisemitism-November-2017-v1-3-2.pdf
+   - text: "Working definition of antisemitism from The International Holocaust Remembrance Alliance (IHRA)"
+      href: https://www.holocaustremembrance.com/working-definition-antisemitism
```

Comment:
```suggestion
     ```

I'm suggesting we remove this resource (despite it being the initial one I came across) because I learned from other sources that they use this rhetoric to preclude support for Palestine.